src/wallet/wallet.h

```diff
@@ -590,7 +629,7 @@ class CWalletTx
     // in place.
     std::set<uint256> GetConflicts() const NO_THREAD_SAFETY_ANALYSIS;
 
-    void SetMerkleBranch(const uint256& block_hash, int posInBlock);
+    void SetMerkleBranch(Status status, const uint256& block_hash, int posInBlock);
```

Comment:
nit: this function is badly named (and has been since 391dff16fe9ace90fc0f3308a5c63c453370e713 _Do not store Merkle branches in the wallet._). Now would be a good time to rename it to `SetTxConf()` or similar.